src/main/java/teammates/common/util/Sanitizer.java

```diff
@@ -215,6 +215,16 @@ public static String sanitizeForCsv(String str){
     private static String trimIfNotNull(String string) {
         return ((string == null) ? null : string.trim());
     }
+
+    /**
+     * Removes leading, trailing, and duplicate internal spaces in the string if it is not null.
+     *
+     * @param string
+     * @return the trimmed string or null (if the parameter was null).
+     */
+    private static String removeExtraSpaceIfNotNull(String string) {
```

Comment:
I think there is no need to create `removeExtraSpaceIfNotNull()` because `removeExtraSpace()` has already handled null pointer.